packages/perseus/src/perseus-types.ts

```diff
@@ -83,21 +83,38 @@ export type PerseusWidgetsMap = {
     [key in `video ${number}`]: VideoWidget;
 };
 
-export type PerseusItem = {
+/**
+ * A PerseusItem is the top-level Perseus data type.
+ */
+export type PerseusItem = StandardItem | MultiItem;
+
+/**
+ * A "StandardItem" is a classic Perseus item. It is rendered by the
+ * `ServerItemRenderer` and the layout is pre-set.
+ */
+export type StandardItem = {
     // The details of the question being asked to the user.
     question: PerseusRenderer;
     // A collection of hints to be offered to the user that support answering the question.
     hints: ReadonlyArray<Hint>;
     // Details about the tools the user might need to answer the question
     answerArea: PerseusAnswerArea | null | undefined;
-    // Multi-item should only show up in Test Prep content and it is a variant of a PerseusItem
-    _multi: any;
     // The version of the item.  Not used by Perseus
     itemDataVersion: Version;
     // Deprecated field
     answer: any;
 };
 
+/**
+ * A "MultiItem" is an advanced Perseus item. It is rendered by the
+ * `MultiRenderer` and you can control the layout of individual parts of the
+ * item.
+ */
+export type MultiItem = {
+    // Multi-item should only show up in Test Prep content and it is a variant of a PerseusItem
+    _multi: any;
+};
```

Comment:
**No op, just a question for myself:**

Since I don't see `MultiItem` being used anywhere in this PR, I assume the main place this is used would be in `webapp` as an imported type, right?

Reply:
Would it ever be possible to fully add types for this?

Reply:
Yes. But these types in this area are complicated and somewhat "stress test" the TypeScript engine.  We found that bringing that typing all the way up to here causes the TypeScript service to crash sometimes. I never figured out how to resolve that.